packages/gene-tools/src/generators/nextjs-app-generator/files/app/pages/index.tsx__tmpl__

```diff
@@ -41,6 +41,7 @@ function HomePage() {
   
 export const getServerSideProps: GetServerSideProps = async ({req}) => {
     const reactQueryClient = reactQueryFactory(() => new QueryClient());
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Maybe, we should add some comment like `TODO: remove this`? In general, I guess, we don't want to encourage usage of eslint-disable right from project start?

Reply:
The only reason it is here, is so that generators integration tests pass (linter).  I'd rather keep this